x-pack/plugins/ingest_manager/server/services/api_keys/index.ts

```diff
@@ -22,8 +22,8 @@ export async function generateOutputApiKey(
       cluster: ['monitor'],
       index: [
         {
-          names: ['logs-*', 'metrics-*'],
-          privileges: ['write'],
+          names: ['logs-*', 'metrics-*', 'events-*', 'metricbeat*'],
```

Comment:
@nchaulet What is `metricbeat*` doing in the list here?

Reply:
I had to add it while demoing the agent last friday, the agent was sending some data to metricbeat* I am going to check which index in details

Reply:
@michalpristas @ph ? ^

Reply:
nothing should be sent to metricbeat*, i will check, maybe some stale metricbeat binary i gave you

Reply:
defaultIndex for elasticsearch output is in form `beat-version-yyyy-mm-dd` if no index is specified. i need to figure out why metricbeat sends data to this index but filebeat is not, configuration of output is the same for both